src/ncp/spinel.h

```diff
@@ -2318,6 +2318,12 @@ typedef enum
      */
     SPINEL_PROP_PARENT_RESPONSE_INFO = SPINEL_PROP_OPENTHREAD__BEGIN + 13,
 
+    SPINEL_PROP_PERFORMANCE_TEST = SPINEL_PROP_OPENTHREAD__BEGIN + 14,
```

Comment:
Adding documentation here? 
What the properties are and their spinel encoding format (and if any required capability)...